py/desitarget/cuts.py

```diff
@@ -788,8 +802,13 @@ def isSTD(gflux=None, rflux=None, zflux=None, primary=None,
         gbright = 16.
         gfaint = 19.
 
-    std &= gaiagmag >= gbright
-    std &= gaiagmag < gfaint
+    if usegaia:
+        std &= gaiagmag >= gbright
+        std &= gaiagmag < gfaint
+    else:
+        gmag = 22.5 - 2.5 * np.log10( gflux.clip(1e-16) )
```

Comment:
I believe that Gaia G-band is better approximated by r than g. So, consider changing this to r fluxes.